UI/volume-control.cpp

```diff
@@ -362,23 +369,24 @@ VolControl::VolControl(OBSSource source_, bool showConfig, bool vertical)
 		meterLayout->setContentsMargins(0, 0, 0, 0);
 		meterLayout->setSpacing(0);
 
-		if (showConfig) {
-			meterLayout->addWidget(config);
-			meterLayout->setAlignment(config, Qt::AlignVCenter);
-		}
 		meterLayout->addWidget(volMeter);
+		meterLayout->addWidget(slider);
 
-		botLayout->setContentsMargins(0, 0, 0, 0);
-		botLayout->setSpacing(0);
-		botLayout->addWidget(mute);
-		botLayout->addWidget(slider);
+		buttonLayout->setContentsMargins(0, 0, 0, 0);
+		buttonLayout->setSpacing(0);
 
-		botLayout->setAlignment(slider, Qt::AlignVCenter);
-		botLayout->setAlignment(mute, Qt::AlignVCenter);
+		if (showConfig) {
+			buttonLayout->addWidget(config);
+		}
+		buttonLayout->addItem(
+			new QSpacerItem(0, 0, QSizePolicy::Minimum,
+					QSizePolicy::MinimumExpanding));
+		buttonLayout->addWidget(mute);
 
 		mainLayout->addItem(textLayout);
-		mainLayout->addWidget(meterFrame);
-		mainLayout->addItem(botLayout);
+		mainLayout->addItem(controlLayout);
+		controlLayout->addItem(buttonLayout);
+		controlLayout->addWidget(meterFrame);
```

Comment:
I would generally expect a sub-layout to have all of its items and widgets added before adding the sub-layout to another layout.
```suggestion
		controlLayout->addItem(buttonLayout);
		controlLayout->addWidget(meterFrame);
		mainLayout->addItem(controlLayout);
```

Reply:
Done